.github/container/test-pax.sh

```diff
@@ -306,19 +317,45 @@ else:
       stacked_p.residual_dropout_prob = self.DROPOUT_PROB
       stacked_p.atten_dropout_prob = self.DROPOUT_PROB
 
+      task_p.train.always_use_train_for_model_init=False
+      task_p.model.report_strict_acc=True
+
       return task_p
 
 EOF
 
 ## Launch
 set -ex
-ENABLE_TE=$ENABLE_TE python -m paxml.main \
+if [[ ${EVALUATE} -ne 0 ]]; then
```

Comment:
nit: can save characters by just exporting ENABLE_TE
```suggestion
export ENABLE_TE=${ENABLE_TE}
if [[ ${EVALUATE} -ne 0 ]]; then
```
and removing it from each python call